src/Components/Facility/ConsultationDetails/index.tsx

```diff
@@ -159,6 +165,16 @@ export const ConsultationDetails = (props: any) => {
             setPatientData(data);
           }
 
+          // Get abha number data
+          const { data: abhaNumberData } = await request(
+            routes.abdm.abhaNumber.get,
+            {
+              pathParams: { abhaNumberId: id ?? "" },
+              silent: true,
+            },
+          );
+          setAbhaNumberData(abhaNumberData);
```

Comment:
Can we move this to `useQuery` outside the `useEffect`?

Reply:
Currently, the `routes.abdm.abhaNumber.get` API depends on patient's external id, so this can't be moved out. The patientId props is undefined in certain cases.